tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/repository/ProgrammeMembershipRepository.java

```diff
@@ -43,6 +43,15 @@ List<ProgrammeMembership> findAllProgrammeMembershipInDescOrderByTraineeId(
       + "WHERE pm.programme.id = :programmeId")
   List<ProgrammeMembership> findByProgrammeId(@Param("programmeId") Long programmeId);
 
+  /**
+   * Get a list of programme memberships.
+   *
+   * @param uuids List of all the programme memberships.
+   * @return a list of programme memberships for all the uuid.
+   */
+  List<ProgrammeMembership> findProgrammeMembershipsByUuidIn(
+      List<UUID> uuids);
```

Comment:
Given there is already something which takes a `Set<UUID>` and returns the same content, is it not possible to reuse that? It feels a little awkward to duplicate the same functionality here.

Reply:
the reason I have it as a List alternative is because we can receive duplicate ids such as if a form A and B are for the same programme, a list allows duplicates where as a Set is unique only

Reply:
Ah, ok. Do you have another PR that I can use to see the whole journey? It still feels odd to me to need to include duplicates, since the ProgrammeMembershipSummaryDTO content will be identical for these, so I would have thought that a single entry could be 'duplicated' as necessary when producing the forms listing, but maybe I'm misunderstanding the process?